qemu/tests/cfg/balloon_service.cfg

```diff
@@ -27,7 +27,7 @@
     polling_interval = 2
     polling_sleep_time = 10
     free_mem_cmd = wmic os get FreePhysicalMemory
-    ratio = 0.5
```

Comment:
On Thu, 2016-05-19 at 23:24 -0700, Cong wrote:
> In qemu/tests/cfg/balloon_service.cfg:
> 
> > @@ -27,7 +27,7 @@
> >      polling_interval = 2
> >      polling_sleep_time = 10
> >      free_mem_cmd = wmic os get FreePhysicalMemory
> > -    ratio = 0.5
>      1. balloon_check.cfg won't be loaded when executing case
>         "balloon_service".

Thanks for comments.
It seems that you're right.
I'll check it again.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHubWeb Bug from
> https://github.com/notifications/beacon/AEzhbeSWYkWj_npLLo9OQwdH-JagyK1Uks5qDVOWgaJpZM4Ii4xv.gif
> 
>